apple-pie/styles/style.css

```diff
@@ -0,0 +1,14 @@
+.title{background-image:url("../images/apple-pie.jpg");
+    background-size: cover;
+    height: 500px;
```

Comment:
Here you could replace the height with a padding of 250px and that would center the image automatically